test/test_ops.py

```diff
@@ -506,39 +506,39 @@ def quote_strs(v):
 
                 if variant in method_or_inplace:
                     fn_template = '''
-                        def _fn(t0{c}{args_annot_kw}):
+                        def _fn(t0{c}{args}):
```

Comment:
Note for review:
This approach is consistent with `test_variant_consistency_jit`
Refer: https://github.com/pytorch/pytorch/pull/58838#discussion_r639805371

Will let other reviewers judge this part as I have limited knowledge about the same.